### ISSUE TITLE
Training-related issues

### ISSUE THREAD

**Issue by lizhang-X** [fundwotsai2001/MuseControlLite#2]
Hello! I'm encountering a graphics memory shortage issue during training. Your paper mentions using a GeForce RTX 3090 with a batch size of 128, but with my GeForce A40, I can only set the batch size to 16. I'm using the same dataset as in your paper, with 47-second segments. Where might I be going wrong? Thank you!
# training hyperparameters
        "GPU_id" : "6,7",

        "train_batch_size": 16,

        "learning_rate": 1e-4,

        "attn_processor_type": "rotary", # "rotary", "rotary_conv_in", "absolute" 

        "gradient_accumulation_steps": 8,

        "max_train_steps": 40000,

        "num_train_epochs": 20,

        "dataloader_num_workers": 16,

        "mixed_precision": "bf16", #["no", "fp16", "bf16"]

        "apadapter": True,

        "lr_scheduler": "constant", # ["linear", "cosine", "cosine_with_restarts", "polynomial", "constant", "constant_with_warmup"]'

        "weight_decay": 1e-2

**Comment by fundwotsai2001** [comment]
Equivalent batch size = gradient accumulation * training batch size

But if using RTX 3090, I used  "train_batch_size": 8, "gradient_accumulation_steps": 16. The setting I uploaded is the record I used when testing on A6000.
